common/wonder_upgrades/wc_wonders_historical_upgrades.txt

```diff
@@ -66,4 +64,23 @@ upgrade_crucible_of_storms = {
 		text_effect_void_inflow = 1
 		text_effect_naga_retinue = 1
 	}
+}
+upgrade_mokrash = {
+	picture = GFX_upgrade_icon_mokrash
+
+	potential = { }
+	allow = { }
+	active = { }
+
+	flags = {
+		is_singular
+	}
+	build_time = 24
+	gold_cost_flat = 400
+	gold_cost_ticking = 1
+	local_modifier = {
+		galleys_perc = -0.10
+	}
+	owner_modifier = {
+	}
 }
```

Comment:
> Mok'rash, a sea giant, calls this island his home, and he is responsible for quite a few sunken ships wanting to pull into the bay.

I don't think it's good idea to add this as an upgrade

1 - There is no way of removing it from wonder, except loot the wonder.
2 - After looting the wonder, anyone can build this upgrade again.

Reply:
maybe replace by Mok'rash's treasury?

Reply:
Do you know of a better alternative, maybe a province modifier? I just like Mok'rash for the easter egg/flavor of the province. He's not ultimately necessary for this branch

Reply:
> 1 - There is no way of removing it from wonder, except loot the wonder.

@MemeWizar, isn't it impossible even via looting?

Reply:
Mok'rash killed by looting in test, and player is able to rebuild, `allow_construction` is non-applicable to upgrades, so Mok'rash is removed